### ISSUE TITLE
LG Gram 17 17Z990 fails to boot

### ISSUE THREAD

**Issue by saxon-viking** [ventoy/Ventoy#1844]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.79

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Both

### Partition Style

GPT

### Disk Capacity

64GB

### Disk Manufacturer

SanDisk

### Image file checksum (if applicable)

Yes.

### What happened?

I get an error message that /efi/xxxx can't be found. It was so small I had trouble reading it and forgot to take a photo with my phone.

 I have tried MBR and GPT methods, Secureboot on and off, same error. I downloaded Ventoy again and checked the checksum. Still didn't work.

I reverted to 1.0.63 and it works flawlessly, currently installing Garuda. Which installed perfectly. An excellent tool.

**Comment by ventoy** [comment]
Which ISO file did you test?

**Comment by saxon-viking** [comment]
Update 24 Sep 2022:
Firstly, to respond to your question, I didn't get past the /efi/missing error to get to any of the ISO's.

I used the same usb stick to try and create a windows 10 recovery CD, which failed, as it always does. Then after that I thought I'd try 1.0.79 Ventoy so I can replicate the errors and take a photo of the screen message. Guess what? It worked flawlessly this time.

I added an NVME M.2 to my pc, then cloned the original HDD to it using Macrium reflect and it failed 3 times. Then it worked once I manually selected partitions. After that I booted into Win10 using boot selection on boot up to verify Win10 was working, before I zapped the old installation.

I used MBR again and install Garuda linux on the same HDD as Win 10. But prior to the installation, I used the live environment on all the iso's and they all worked.

So I have no idea what the problem was, and this can be closed now. Thanks
